docker-compose.yml

```diff
@@ -38,7 +37,6 @@ services:
     command: scout --host mongodb --demo serve --host 0.0.0.0
     volumes:
       - ./scout:/home/worker/app/scout
-      - ./volumes/scout/data:/home/worker/data
```

Comment:
👍 We are not really using it there as it is now. Devs should likely map their own volumes for both really, one for a live scout dir, one for a live persistent database. We could make an example of that in containers/ at some point, but not this PR.